pyproject.toml

```diff
@@ -34,6 +34,7 @@ dev = [
     "ruff",
     "pytest-asyncio>=0.25.3",
     "jiwer>=3.1.0",
+    "pyav>=14.1.0",
```

Comment:
if we are pulling this in, we can ignore the `codecs` optional dep?

Reply:
this is in the dev dependencies